test/tests/vsoClientTests.ts

```diff
@@ -0,0 +1,9 @@
+import assert = require('assert');
```

Comment:
Is this a placeholder? If we aren't using it for now we can remove until we need it.